### PR TITLE
SONARPY-1380: Typeshed third party protobufs serialization should remove older version of protobufs

### PR DESCRIPTION
**Pull request by joke1196**

### CODE REVIEW

File: its/ruling/src/test/resources/expected_extended/airflow/python-S930.json
Comment (joke1196) [inline its/ruling/src/test/resources/expected_extended/airflow/python-S930.json:1]: This was an FP linked to older stubs.

File: python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java [hunk 1]
```diff
@@ -542,20 +542,21 @@ public void modules_whose_name_differ_by_capitalization_only() {
...
-    SymbolImpl baseServer = (SymbolImpl) socketServer.get("BaseServer");
-    assertThat(baseServer.validForPythonVersions()).containsExactlyInAnyOrder("27");
+    assertThat(socketServer).isEmpty();
 
     // Python 2, import socketserver
     // SONARPY-1359: socketserver is missing from the standard library stubs in Python 2, but it is indirectly serialized through the stubs serialized by the mypy importer
     socketServer = symbolsForModule("socketserver");
     assertThat(socketServer).isNotEmpty();
+    SymbolImpl baseServer = (SymbolImpl) socketServer.get("BaseServer");
+    assertThat(baseServer.validForPythonVersions()).isEmpty();
```
Comment (joke1196) [inline python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java:552]: Not 100% sure about this one as we do find the symbol but it is not valid for 27

File: python-frontend/typeshed_serializer/tests/test_serializers.py [hunk 2]
```diff
@@ -86,3 +84,5 @@ def test_save_merged_symbols():
         ts_serializer.serialize_merged_modules()
         assert symbols.save_module.call_count == 1
         assert symbols.save_module.mock_calls[0].args[0] == merged_module_symbol
+
+
```
Comment (guillaume-dequenne-sonarsource) [inline python-frontend/typeshed_serializer/tests/test_serializers.py:88]: These extra whitespaces are unneeded.

File: python-checks/src/test/resources/checks/duplicateArgument.py [hunk 3]
```diff
@@ -82,8 +82,8 @@ def rectangle((top, left), (width, height)): ...
 
 def function_from_typeshed():
```
Comment (guillaume-dequenne-sonarsource) [inline python-checks/src/test/resources/checks/duplicateArgument.py:83]: I have the feeling this commit could be left out of this PR and merged separately, as it is unrelated to the ticket.

**Review by guillaume-dequenne-sonarsource — CHANGES_REQUESTED** [review]
Looks good!
Some minor changes requested.

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM. Probably you can squash the main changes and the ruling updates.

**Pull request merged**